tests/phpunit/api/v3/PaymentTest.php

```diff
@@ -52,6 +52,8 @@ public function setUp() {
 
     $this->_apiversion = 3;
     $this->_individualId = $this->individualCreate();
+    $config = CRM_Core_Config::singleton();
+    $config->userPermissionClass->permissions = array();
```

Comment:
Minor aside - you can condense this to

CRM_Core_Config::singleton()->userPermissionClass->permissions = array();